spec/support/shared/updated_content.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.shared_examples 'updated content' do |name|
+  let(:mod) { Training::Module.by_name(:alpha) }
+  let(:content) { mod.page_by_name(name) }
+
+  describe '#edited?' do
+    context 'when preview' do
+      # make Rails.application.preview? return true
+      it do
+        allow(Rails.application).to receive(:preview?).and_return(true)
+        expect(content.edited?).to be true
+      end
+    end
+
+    context 'when delivery' do
+      it do
+        create(:module_release, first_published_at: Time.zone.local(2023, 1, 1))
+        expect(content.edited?).to be true
+      end
+
+      it do
+        create(:module_release, first_published_at: Time.zone.local(3_099, 1, 1))
+        expect(content.edited?).to be false
```

Comment:
`expect(content).to_not be_edited` RSpec allows you to turn predicates of your own creation into sentences.